src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Policy Insights Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure policy insights operations.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.1.0")]
+[assembly: AssemblyVersion("2.0.2.0")]
```

Comment:
`AssemblyVersion` Remains the same, AssemblyFileVersion needs to be updated

Reply:
Done.